extensions/binding/org.eclipse.smarthome.binding.mqttbroker/ESH-INF/i18n/mqttbroker_en.properties

```diff
@@ -0,0 +1,4 @@
+offline.notextualconfig=The textual configured broker with the name {0} doesn't exist anymore
+offline.dyninsteadoftextual=A binding generated broker connection was found instead of a textual configured one for the broker name: {0}
+offline.textualinsteadofdny=A textual configured binding was found instead of a generated broker connection for the broker name: {0}
```

Comment:
Passt nicht ganz, oder:
eng: textual configured binding
ger: Konfigurationsdatei konfigurierte Verbindung

Reply:
That's what came to my mind when I translated it. If you have a better suggestion please let me know. I believe it's not that easy to express "textual configured" in German.

Reply:
maybe "textuell konfiguriert"

Reply:
> maybe "textuell konfiguriert"

Why not :+1: 
But it's also about "binding" vs. "Verbindung". Isn't "binding" in ESH something completely different to "connection"?

Reply:
Ah, jetzt verstehe ich das Problem.
Some of the connections are configured by this binding, some are configured by the mqtt.cfg file. The whole purpose of this binding is actually to serve as a gui for configuring MQTT connections. Therefore "A binding generated broker connection" does make sense, but is at the same time confusing, because "binding" is an English word.

Reply:
> some are configured by the mqtt.cfg file

And some are programatically added through ´MqttService.addBrokerConnection`...